omero/sysadmins/unix/install-web/install-nginx.txt

```diff
@@ -282,10 +244,6 @@ using :property:`omero.web.wsgi_args`:
 EXPERIMENTAL: Gunicorn advanced configuration
 ---------------------------------------------
 
-.. note:: Experimental configurations are not ready for production use.
```

Comment:
I initially remove the EXPERIMENTAL but rolled back the changes and forgot to roll back the notes